setup.py

```diff
@@ -164,16 +248,42 @@ def runtest(self):
     license="BSD",
     long_description="",
     name="cellprofiler",
+    options = {
+        "py2exe": {
+            "dll_excludes": [
+                "iphlpapi.dll",
+                "jvm.dll",
+                "msvcr90.dll",
+                "msvcm90.dll",
+                "msvcp90.dll",
+                "nsi.dll",
+                "winnsi.dll"
+                ],
+            "excludes": [
+                "Cython",
+                "IPython",
+                "pylab",
+                "Tkinter"
+                ],
+            "includes": [
+                "h5py", "h5py.*",
+                "lxml", "lxml.*",
+                "scipy.io.matlab.streams", "scipy.special", "scipy.special.*",
+                "scipy.sparse.csgraph._validation",
+                "skimage.draw", "skimage._shared.geometry", 
+                "skimage.filters.rank.*",
+                "sklearn.*", "sklearn.neighbors", "sklearn.neighbors.*",
+                "sklearn.utils.*", "sklearn.utils.sparsetools.*",
+                "zmq", "zmq.utils", "zmq.utils.*", "zmq.utils.strtypes"
+                ],
+            "packages": packages,
```

Comment:
Good question. I think it's expedient to leave tutorial out (what if there's a Python package named "tutorial"?). IMHO there's not enough value at the moment to including it.